issue_template.md

```diff
@@ -21,6 +21,8 @@
 -   Are there other R packages that accomplish the same thing? If so, how does
 yours differ or meet [our criteria for best-in-category](https://github.com/ropensci/onboarding/blob/master/policies.md#overlap)?
 
+-   If you had made a successful pre-submission enquiry, please copy below the link to the corresponding issue.
```

Comment:
How about: If you had made a successful pre-submission enquiry, please paste the link to the corresponding issue _(or forum post, twitter thread, or other discussion)_.

Reply:
Great point. Should we also suggest to tag the editor in case of a private discussion 🤔

Reply:
Sure, let's see how to do this with minimum wordiness:

>If you made a pre-submission enquiry, please paste the link to the corresponding issue, forum post, or other discussion, or `@tag` the editor you contacted.

Reply:
perfect! 😸